Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -11,16 +20,26 @@ namespace Fuse.Internal
 	struct MiniList<T> : IList<T> where T : class
 	{
 		object _list;
+		ObjectList<T> AsList { get { return (ObjectList<T>)_list; } }
+		
+		MiniListMode _mode = MiniListMode.Empty;
```

Comment:
This makes this container twice as large, and no longer very "Mini"... Why do we need this?